node/tests/ServerModules.test.ts

```diff
@@ -717,228 +691,306 @@ describe("Server Module Tests", () => {
-                    client,
-                    name,
-                    [
-                        { type: "TAG", name: "author_id" },
-                        { type: "TAG", name: "author_ids" },
-                        { type: "TEXT", name: "title" },
-                        { type: "TEXT", name: "name" },
-                    ],
-                    {
-                        dataType: "HASH",
-                        prefixes: ["author:details:", "book:details:"],
-                    },
-                ),
-            ).toEqual("OK");
-
-            // create a duplicating index - expect a RequestError
-            try {
-                expect(
-                    await GlideFt.create(client, name, [
-                        { type: "TEXT", name: "title" },
-                        { type: "TEXT", name: "name" },
                     ]),
-                ).rejects.toThrow();
-            } catch (e) {
-                expect((e as Error).message).toContain("already exists");
-            }
-
-            // create an index without fields - expect a RequestError
-            try {
-                expect(
-                    await GlideFt.create(client, uuidv4(), []),
-                ).rejects.toThrow();
-            } catch (e) {
-                expect((e as Error).message).toContain(
-                    "wrong number of arguments",
+                ).toEqual("OK");
+
+                const before = await client.customCommand(["FT._LIST"]);
+                expect(before).toContain(index);
+
+                // DROP it
+                expect(await GlideFt.dropindex(client, index)).toEqual("OK");
+
+                const after = await client.customCommand(["FT._LIST"]);
+                expect(after).not.toContain(index);
+
+                // dropping the index again results in an error
+                try {
+                    expect(
+                        await GlideFt.dropindex(client, index),
+                    ).rejects.toThrow();
+                } catch (e) {
+                    expect((e as Error).message).toContain(
+                        "Index does not exist",
+                    );
+                }
+            });
+
+            it("FT.INFO ft.info", async () => {
+                client = await GlideClusterClient.createClient(
+                    getClientConfigurationOption(
+                        cluster.getAddresses(),
+                        protocol,
+                    ),
                 );
-            }
 
-            // duplicated field name - expect a RequestError
-            try {
+                // TODO use FT.LIST when it is done
+                const indices = (await client.customCommand([
+                    "FT._LIST",
+                ])) as GlideString[];
+
+                // check that we can get a response for all indices (no crashes on value conversion or so)
+                for (const idx in indices) {
+                    await GlideFt.info(client, idx);
```

Comment:
this test step is very random and could be flaky.

Reply:
Any failure is appreciated.
But it would be hard to reproduce or get any helpful info to debug it.

Reply:
a good test should reproducable

Reply:
Should I remove this there and from java tests too?